src/actions/patchers.ts

```diff
@@ -649,28 +649,62 @@ export const setInitialPresetOnRemoteInstance = (instance: PatcherInstanceRecord
 		}
 	};
 
-export const sendInstanceMessageToRemote = (instance: PatcherInstanceRecord, inportId: string, value: string): AppThunk =>
-	(dispatch) => {
-		const values = value.split(" ").reduce((values, v) => {
-			const fv = parseFloat(v.replaceAll(",", ".").trim());
-			if (!isNaN(fv)) values.push({ type: "f", value: fv });
-			return values;
-		}, [] as OSCArgument[]);
+export const triggerSendInstanceInportMessage = (instance: PatcherInstanceRecord, port: MessagePortRecord): AppThunk =>
+	async (dispatch) => {
+		try {
+
+			const dialogResult = await showTextInputDialog({
+				text: `Send Inport message to ${port.name}`,
+				actions: {
+					confirm: { label: "Send" }
+				},
+				validate: (value: string) => {
+					if (!value.length) return "Please provide a value";
+					const values = value.trim().replaceAll(",", ".").split(" ");
+					if (values.find(v => isNaN(parseFloat(v)))) return "Please provide a single or multiple numbers separated by a space";
```

Comment:
this might be better worded `please provide a number or multiple numbers separated by spaces`